Source/WebCore/xml/XPathGrammar.y

```diff
@@ -260,8 +260,10 @@ NodeTest:
     |
     PI '(' LITERAL ')'
     {
-        String literal = adoptRef($3);
-        $$ = new WebCore::XPath::Step::NodeTest(WebCore::XPath::Step::NodeTest::ProcessingInstructionNodeTest, literal.stripWhiteSpace());
+        auto stringImpl = adoptRef($3);
+        if (stringImpl)
+            stringImpl = stringImpl->stripLeadingAndTrailingCharacters(deprecatedIsSpaceOrNewline);
+        $$ = new WebCore::XPath::Step::NodeTest(WebCore::XPath::Step::NodeTest::ProcessingInstructionNodeTest, stringImpl.get());
```

Comment:
Maybe we can move the stripping inside the `NodeTest` constructor.

Reply:
I couldn't really find the constructor. `ProcessingInstructionNodeTest` only has a couple hits.